src/EventStore.Core/Services/RequestManager/RequestManagementService.cs

```diff
@@ -167,7 +167,15 @@ public class RequestManagementService :
 		
 		
 		public void Handle(SystemMessage.StateChangeMessage message) {
-			//TODO(clc): if we have become resigning leader should all requests be actively disposed?
+			
+			if (_nodeState == VNodeState.Leader && message.State is not VNodeState.Leader or VNodeState.ResigningLeader) {
```

Comment:
do we still want the change in this file? the node is just about to go offline for truncation so this might just be extra noise now